src/modules/commander/commander.cpp

```diff
@@ -2381,6 +2390,8 @@ Commander::run()
 			}
 		}
 
+		_last_sp_man = sp_man;
```

Comment:
Changing the position here could have unexpected side-effects.

Reply:
Agreed, too risky for the release.

Reply:
Alright, let's review this post v1.8.0 then. In some cases the way some of these features actually function has drifted from what I think may have been intended.